src/ShaderManager.js

```diff
@@ -65,8 +62,11 @@ class ShaderManager {
         }
 
         const definesText = `${defines.join('\n')}\n`;
-        const vsFullText = definesText + vertexShaderText;
-        const fsFullText = definesText + fragmentShaderText;
+
+        /* eslint-disable global-require */
+        const vsFullText = definesText + require('raw-loader!./shaders/sprite.vert');
+        const fsFullText = definesText + require('raw-loader!./shaders/sprite.frag');
+        /* eslint-enable global-require */
```

Comment:
I'm following the pattern from [scratch-storage@8dddd1f](https://github.com/LLK/scratch-storage/blob/8dddd1f8e2554e8f1726c5f768dd7dbc1b06b8b5/src/BuiltinHelper.js#L24).  The issue is that `tap` runs without a compile step, so it doesn't know what to do with `require('raw-loader!...')`.